onadata/libs/utils/logger_tools.py

```diff
@@ -336,6 +336,10 @@ def save_attachments(xform, instance, media_files, remove_deleted_media=False):
             xform.instances_with_osm = True
             xform.save()
         filename = os.path.basename(f.name)
+        # Validate Attachment file name length
+        if len(filename) > 99:
```

Comment:
Based on the error message, should the number in the condition be 100 instead of 99? At the moment, if the size of the filename is 100, the error message will be triggerd, no?

Reply:
Made the change